app/views/layouts/application.html.erb

```diff
@@ -8,27 +8,45 @@
   <%= stylesheet_link_tag('application', media: 'all') %>
 <% end %>
 
+<% content_for :header_class do "with-proposition" end %>
```

Comment:
Instead of using do / end on a single line you can use `{ }`, they are preferred when the block is in a single line